SKILL.tsv

```diff
@@ -3187,38 +3187,38 @@ SKILL_20150401_003186	{memo X}Change to Testudo formation. All troops move in a
 SKILL_20150401_003187	{memo X}Attack - #{CaptionRatio}#%{nl}Defense increase as much as increased attack
 SKILL_20150401_003188	{memo X}Change direction of formation
 SKILL_20150401_003189	{memo X}The direction you face become front
-SKILL_20150401_003190	Attack speed increases while defense decreases per damage received.
+SKILL_20150401_003190	$Increases your attack speed and attack when you take damage, but decreases defense.
```

Comment:
Suggest:
"When you take damage, increases attack speed and physical attack while decreasing defense."